javascript/javascript1/week3/note.js

```diff
@@ -0,0 +1,46 @@
+//NOnoN0nOYes (Note taking app)
+
+//Save a note
+
+const notes=[]
+
+function saveNote( content, id){
+    const note = { content, id };
+   notes.push(note);
+}
+saveNote( "Pick up groceries", 1);
+saveNote( "Do laundry", 2);
+
+console.log(notes);
+
+//Get a note
+
+function getNote(id) {
+    const findNote = notes.find((note) => note.id === id);
+    return findNote;
+  }
+  
+  const firstNote = getNote(1);
+  console.log(firstNote);
+
+  //Log out notes
+
+  function logOutNotesFormatted() {
+    notes.forEach((note) => {
+      console.log(`The note with id: ${note.id}, has the following note text: ${note.content}`);
+    });
+  }
+  
+  logOutNotesFormatted();
+
+  //Set Reminders for Notes
+
+  function setReminder(id, reminderTime) {
+    const note = notes.find((note) => note.id === id);
```

Comment:
You've switched a bit between for loops and higher-order functions such as `forEach` and `find`. How did you pick which approach to use? 😄 

Can you briefly explain what `find` and `=>` does? 🔍

Reply:
I pick find and forEach method in the list of the mdn website. I found out forEach is easier to write compare for :)
find is a method to recall the first element in which condition suits it. F.X,  notes.find(note) means find the note in the array notes. But we need a condition to find properly so we used "=>".